frontend/src/preload.js

```diff
@@ -1,2 +0,0 @@
-/* eslint no-undef: 0 */
-__webpack_public_path__ = `${window.Sonarr.urlBase}/`;
```

Comment:
I think you need to keep preload to get the fuse worker to load properly when `urlBase` is defined.  IIRC it's something to do with the `__webpack_public_path__` not getting picked up correctly otherwise due to the js load order.

I had to restore it in #3430 after it was dropped in the previous webpack PR.

Reply:
I recall needing this to make monaco worker load properly. But I'm not planning on merging this in anytime soon. I just need it for debugging something at catalina :)

Reply:
Yeah, I used the inline option in the worker-loader so it doesn't need to pull in a separate worker js, it just loads from the bundle.